data/behavior_data.c

```diff
@@ -2661,7 +2662,7 @@ const BehaviorScript bhvBowserSubDoor[] = {
     BEGIN(OBJ_LIST_SURFACE),
     OR_LONG(oFlags, (OBJ_FLAG_ACTIVE_FROM_AFAR | OBJ_FLAG_UPDATE_GFX_POS_AND_ANGLE | OBJ_FLAG_UCODE_LARGE)),
     LOAD_COLLISION_DATA(ddd_seg7_collision_bowser_sub_door),
-    SET_FLOAT(oDrawingDistance, 20000),
+    SET_FLOAT(oDrawingDistance,   20000),
```

Comment:
Lots of formatting changes - are these necessary?

Reply:
There were only 2 of those changes. Those were for consistency with the other places oDrawingDistance and oCollisionDistance were already used together in the file.

Reply:
Please revert the formatting changes, as they'll cause merge conflicts in the future if base decomp incorporates them as well.